yarn.lock

```diff
@@ -2162,11 +2162,6 @@
     stylelint-prettier "^2.0.0"
     stylelint-scss "^4.0.0"
 
-"@shopify/typescript-configs@^5.0.0":
```

Comment:
When running `yarn` on this branch, I'm also getting some other `yarn.lock` updates. Maybe just do a double-check that there aren't any other dep updates that need to be made to the lock file

Reply:
d'oh. fixed!